---
string  if_ip_address   # network interface's IP address
string  if_ip_mask      # network interface's IP mask
string  if_ip_gateway   # network interface's IP gateway
string  dev_ip_address  # device IP address
string  dev_ip_mask     # device IP mask
string  dev_ip_gateway  # device IP gateway
